data/ScottDeev.json

```
@@ -0,0 +1,24 @@
+{
+  "name": "Ameh Elijah Uma-ojo",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "FRONTEND DEV - JavaScript || React || Firebase || TailwindCSS",
+  "avatar": "https://github.com/scottDeev.png",
+  "links": [
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/ElijahLauren",
+      "icon": "FaTwitter"
+    },
+    {
+      "name": "Follow me on LinkedIn",
+      "url": "https://www.linkedin.com/in/abhishek-gupta-95275b233",
+      "icon": "FaLinkedin"
+    },
+    {
+      "name": "Follow me on Github",
+      "url": "https://github.com/scottDeev" ,
```

There's an extra space here (not sure if it'll get fixed automatically).
```suggestion
      "url": "https://github.com/scottDeev",
```